packages/auth/src/config/app.ts

```diff
@@ -32,5 +34,6 @@ export const Config = {
   ]),
   resourceServerDomain: envString('INTERACT_URL', 'http://localhost:3004'),
   authServerDomain: envString('DOMAIN', 'http://localhost:3006'), // TODO: replace this with whatever frontend port ends up being
-  waitTime: envInt('WAIT', 5)
+  waitTime: envInt('WAIT', 5),
```

Comment:
we should add units to this variable. Difficult to know what it represents when glancing